pkg/arvo/lib/metadata-json.hoon

```diff
@@ -0,0 +1,64 @@
+/-  *metadata-store
+|%
+++  associations-to-json
+  |=  =associations
+  =,  enjs:format
+  ^-  json
+  %+  frond  %metadata-initial
+  %-  pairs
+  %+  turn  ~(tap by associations)
+  |=  [[=group-path =resource] =metadata]
+  ^-  [cord json]
+  :_  (metadata-to-json metadata)
+  %-  crip
+  ;:  weld
+      (trip (spat group-path))
+      (trip app-name.resource)
+      (trip (spat app-path.resource))
+  ==
```

Comment:
This is going to look like `/group-pathapp-name/app-path`.

Even with the `app-name` more cleanly separated from the `group-path`, are we sure it's fine to just concat them into a string like this? I don't think you have any guaranteed on where one part ends and the other begins, if you're a client trying to parse these out.

If these are only going to be used in contexts where you already know the `group-path`, `app-name` and `app-path`, and you only care about doing lookup for those (that you know) specifically, then maybe this is fine... But it's certainly weird!

Reply:
I'll make this a better data structure. Good critique.

Reply:
Fixed the structure to include the data in a separable format.

Reply:
The "Even if ..." from above still applies, I'm pretty sure. Still sub-optimal, but functional enough?